src/components/ServiceContainers/ServiceContainerLoading.tsx

```diff
@@ -38,7 +38,7 @@ interface IServiceContainerLoadingProps extends IServiceContainerProps {
  * @param data - Real data to be displayed when its fully loaded
  * @param loading - True if a request is pending, false when a request is successfully finished or when a request resulted in error
  * @param error - Error description when data loading was not successful
- * @param loadingDelayMilliseconds - Waiting time before loading component gets rendered
+ * @param loadingStateDelay - Waiting time before loading component gets rendered
```

Comment:
Add information about milliseconds (see jira and chat comments). Make sure to provide exactly the same way also in `useServiceContainer`, so hook and UI component delay related arguments (name, tsdoc) will be matching.

Reply:
I decided for: `loadingStateDelayMs`.